### ISSUE TITLE
Design | Profile | Midpoint Review - Accessibility Feedback - Profile Team - Bad Address Indicator

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#43095]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
- Thank you to @SKasimow for the staging user! Getting to see it in staging was super helpful, and not something we normally get at a midpoint review.

## Feedback
- **Must** = the feedback must be applied
- **Should** = the feedback is best practice
- **Consider** = suggestions/enhancements

**Must:**
I have a few concerns with the "edit it" link/button in the "Please confirm your address" alert.

![please-confirm-your-address](https://user-images.githubusercontent.com/6844775/174391115-16c2c87e-f712-4cf1-aca1-8f1f742c9881.png)

The `Edit Address` and `edit it` links are really buttons styled to look like links, which is not a practice we encourage at this point. Instead, we encourage teams to practice material honesty --- buttons look like buttons and do actions, links look like links and go places.

There are a few reasons for this, but one easy one is for voice command software users, who may not be able to use a keyboard or mouse and rely on speaking commands to their computer. One common command is `Click link`, at which point the software inserts numbers next to each link on the page. The user then speaks the number that corresponds to the link they want, eg. "Click 7."

That interaction breaks down when a link isn't actually a link but is instead a `<button>`, as is the case with these. Users are left to guess whether something is broken on the page, or whether it's really a button, or whether something else might be going on. Frustrating experience all around.

From our conversation in the meeting it sounds like `Edit Address` is out of scope for this iteration. But for `edit it`, the action should be identifiably a button.

I'm also not sure how much value there is to having both the `edit it` button and the `Edit Address` button in such close proximity and with the same function. You might be able to get away with removing the `edit it` button entirely, which is the probably easiest solution! That might be worth exploring in your research.

**Should:**
- Just to keep piling on with the `edit it` button... 
  - The purpose of the `edit it` button is fairly clear from context. But screen reader users often navigate the page by skipping from one interactive element to another, so the button text may be announced without that context. On a page with lots of things that can be edited, "edit it" leaves the user potentially guessing about what it is they're going to be changing. More descriptive text would be better.
  - Another voice command interaction that some software supports is speaking a command like "Click edit it." That's a little difficult to say out loud and may require the user to be extra deliberate in how they enunciate. More descriptive text would help with that too.
- As the coded version of this is finalized, it's worth taking a moment to make a document outline for all of your alert headings. You mentioned this in the meeting (thank you!) so it's already on your mind, but I'll remind you anyway! Headings define sections and sub-sections of the page, so anything that follows a heading should directly relate to that heading. Between the existing headings on the page and the ones that are being added via these alerts, make sure content is sectioned off in a way that makes sense. I'm happy to chat more about this one-on-one if there are any fuzzy situations you run into.

**Consider:**

## Platform directions
- Update "Issue Title"
- Link to collab cycle Request epic
- Add your feedback
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review QA Only**: VFS PM, yourself (optional)

**Comment by SKasimow** [comment]
Hi @briandeconinck - thank you for your feedback. The “edit address” section is out-of-scope for BAI (it's actually part of our address validation flow), so we won’t be making  changes to this section right now. We do plan to re-visit this in the future and appreciate your offer to assist.

Can we close this ticket? Or, we can  park in the "icebox" for future?

cc/ @briandeconinck

**Comment by SKasimow** [comment]
Hi @cgednalske - think this can also be added to accessibility document (I've also added the 508-accessbility tag).


+ @Samara-Strauss @andaleliz

**Comment by cgednalske** [comment]
@SKasimow perfect! We have that concern documented, but I totally forgot about the related ticket. It's been added in the [address validation section of the doc](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/accessibility/accessibility-audit-updates.md#address-validation) 😃

**Comment by Samara-Strauss** [comment]
Moving this to Backlog since this since we are not yet ready to work on address validation updates.

**Comment by SKasimow** [comment]
We have an accessibility ticket drafted for address validation #47272

**Comment by SKasimow** [comment]
If no objections, I'm closing this as this feedback is being addressed in #48537
